src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -44,6 +53,30 @@ export const CreateTransaction = (props: CreateProps) => {
     [StellarSdk.MemoNone]: "",
   };
 
+  const fetchIfFederationAddress = async () => {
+    const { toAccountId } = formData;
+    if (isFederationAddress(toAccountId)) {
+      setFederationAddressFetchStatus(ActionStatus.PENDING);
+      try {
+        const response = await FederationServer.resolve(toAccountId);
+        setFederationAddressFetchStatus(ActionStatus.SUCCESS);
+        onInput({
+          ...formData,
+          federationAddress: response.account_id,
+        });
```

Comment:
You have to check for memos and memo types too, since some people implement federation with one account for all, with account info in the memo. See also https://www.stellar.org/developers/guides/concepts/federation.html#federation-response